scripts/openzfs-tracking.sh

```diff
@@ -221,13 +222,17 @@ do
 	ZFSONLINUX_REGEX="^(openzfs|illumos)+.*[ #]+$OPENZFS_ISSUE[ ,]+*.*"
 
 	# Commit exceptions reference this Linux commit for an OpenZFS issue.
-	EXCEPTION=$(grep -E "^$OPENZFS_ISSUE.+" "$ZFSONLINUX_EXCEPTIONS")
+	EXCEPTION=$(grep -E "^$OPENZFS_ISSUE\s" "$ZFSONLINUX_EXCEPTIONS")
 	if [ -n "$EXCEPTION" ]; then
 		EXCEPTION_HASH=$(echo $EXCEPTION | cut -f2 -d' ')
+		EXCEPTION_COMMENT=$(echo $EXCEPTION | cut -d' ' -f3-)
 		if [ "$EXCEPTION_HASH" == "-" ]; then
 			ZFSONLINUX_HASH="-"
 			ZFSONLINUX_STATUS=$STATUS_NONAPPLICABLE
 			ZFSONLINUX_STATUS_TEXT=$STATUS_NONAPPLICABLE_TEXT
+			if [ -n "$EXCEPTION_COMMENT" ]; then
+			OPENZFS_DESC="$OPENZFS_DESC</br><b>comment: $EXCEPTION_COMMENT</b>"
```

Comment:
Missing tab.  But I like the idea of making the comment visible in the web page.